tests/client/amo/helpers.js

```diff
@@ -12,6 +12,7 @@ export const fakeAddon = {
     id: 123,
     license: { name: 'tofulicense', url: 'http://license.com/' },
     version: '2.0.0',
+    files: [],
```

Comment:
Ha. How did this work before? I just added it too. https://github.com/mozilla/addons-frontend/pull/1586/files#diff-2e76cee7f42cc26c94ef51183f1b5f86R16